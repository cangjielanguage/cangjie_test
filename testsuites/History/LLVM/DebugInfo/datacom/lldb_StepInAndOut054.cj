// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_StepInAndOut054_expected.txt ../basecase/021_debug_file.cj
// EXEC: cjc -g 021_debug_file.cj
// EXEC: python3 DatacomTest.py lldb_StepInAndOut054_expected.txt



