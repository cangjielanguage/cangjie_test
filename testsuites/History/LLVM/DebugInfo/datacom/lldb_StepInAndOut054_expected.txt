lldb,lldb 021_debug_file,021_debug_file
b,b 8,021_debug_file.cj:8
r,r,-> 8
s,s,-> 14
s,s,-> 18
s,s,-> 22
s,s,-> 26
s,s,-> 30
s,s,-> 34
s,s,-> 38
s,s,-> 42
s,s,-> 46
s,s,-> 51
s,s,-> 55
s,s,-> 59
s,s,-> 63
s,s,-> 67
s,s,-> 71
s,s,-> 75
s,s,-> 79
s,s,-> 83
s,s,-> 87
s,s,-> 92
finish,finish,-> 87
finish,finish,-> 83
finish,finish,-> 79
finish,finish,-> 75
finish,finish,-> 71
finish,finish,-> 67
finish,finish,-> 63
finish,finish,-> 59
finish,finish,-> 55
finish,finish,-> 51
finish,finish,-> 46
finish,finish,-> 42
finish,finish,-> 38
finish,finish,-> 34
finish,finish,-> 30
finish,finish,-> 26
finish,finish,-> 22
finish,finish,-> 18
finish,finish,-> 14